src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -383,16 +394,22 @@ add_slope_flags <- function(df) {
 # }
 ```
 
+add large anomaly flag function
+```{r}
+add_large_anomaly_flag <- function(df) {
+  df <- df %>% 
+    mutate(flag_binary = ifelse((is.na(flag) | flag == "24hr anomaly flag"), 0, 1),
```

Comment:
Add comments here to clarify what each step is doing, please.
Suggestion: use dplyr::if_else() instead of base-r ifelse().

```suggestion
    # add an indication where there is a flag, except when the flag is the 24h anomaly.
    mutate(flag_binary = if_else((is.na(flag) | flag == "24hr anomaly flag"), 0, 1),
```